src/assets/svg/carrot-down.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="24" height="14" viewBox="0 0 24 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M2.59338 0.859505C2.00011 0.266233 1.03823 0.266233 0.444954 0.859504C-0.148318 1.45278 -0.148318 2.41466 0.444954 3.00793L10.8762 13.4392C11.0006 13.5636 11.1411 13.6618 11.2911 13.7341C11.8672 14.0471 12.603 13.96 13.0902 13.4728L23.555 3.00795C24.1483 2.41468 24.1483 1.45279 23.555 0.859522C22.9618 0.26625 21.9999 0.26625 21.4066 0.859522L12 10.2661L2.59338 0.859505Z" fill="white"/>
+<svg width="20" height="10" viewBox="0 0 24 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In general, I prefer removing the `fill=` from SVGs so we can set the color using CSS. That makes it possible to re-use SVGs and give it different color treatments for hover states. Also, when we set color using CSS, we can use variables that are easier to read and update.